DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -0,0 +1,210 @@
+            return day;
+        }
+
+        public string GetDayEndDate()
+        {
+            string day = EndDateFld.GetAttribute("value");
+            return day.Substring(0, 2);
+        }
+
+        public int GetYearStartDate()
+        {
+            string year = YearDropDown.GetAttribute("value");
+            int yearBeforeCalculate = Convert.ToInt32(year);
+            return yearBeforeCalculate;
+        }
+
+        public int GetYearEndDate()
+        {
+            string year = EndDateFld.GetAttribute("value");
+            int yearAfterCalculate = Convert.ToInt32(year.Substring(6, 4));
+            return yearAfterCalculate;
+        }
+
+        public void ClickOnYearDropDown()
+        {
+            YearDropDown.Click();
+        }
+
+        public void ClickOnYear2024()
+        {
+            Year2024.Click();
+        }
+
+        public void ClickOnYear2028()
+        {
+            Year2028.Click();
+        }
+
+        public void SelectDay(string day)
+        {
+            var selectDayDropDown = DayDropDown;
+            var selectDayDropDownElement = new SelectElement(selectDayDropDown);
+            selectDayDropDownElement.SelectByText(day);
+        }
+
+        public void SelectMonth(string month)
+        {
+            var selectMonthDropDown = MonthDropDown;
+            var selectMonthDropDownElement = new SelectElement(selectMonthDropDown);
+            selectMonthDropDownElement.SelectByText(month);
+        }
+
+        public void SelectYear(string year)
+        {
+            var selectYearDropDown = YearDropDown;
+            var selectYearhDropDownElement = new SelectElement(selectYearDropDown);
+            selectYearhDropDownElement.SelectByText(year);
+        }
+
+        public string StartDateMonth
+        {
+            get
+            {
+                return MonthDropDown.GetAttribute("value");
+            }
+
+            set
+            {
+                if (value == "360")
+                {
+                    FinancialYear360Days.Click();
+
+                    // $"{day}/{month}/{yeat}"
+                    return;
+                }
+
+                if (value == "365")
+                {
+                    FinancialYear365Days.Click();
+                    return;
+                }
+
+                throw new Exception("Invalid financial year value");
+            }
+        }
+
+        public string GetMonthEndDateText()
+        {
+            string EndDate = EndDateFld.GetAttribute("value");
+            return EndDate.Substring(3, 2);
+        }
+
+        public string monthNumberParseToText()
+        {
+            string numberStr = GetMonthEndDateText();
+            int number;
+
+            bool isParsable = Int32.TryParse(numberStr, out number);
+
+            if (isParsable)
+                return CultureInfo.InvariantCulture.DateTimeFormat.GetMonthName(number);
+            else
+                return null;
```

Comment:
If something wrong happens, it's better to throw an exception.

```suggestion
                thow new Exception($"Invalid month number: {number}");
```